style/themes/lcars.css

```
@@ -2055,3 +2069,11 @@ td.highlight {
 #dns .col-md-11 {
   width: 84%;
 }
+
+/*** add style to the information/about dropdown ***/
+
+.user-header,
+.user-footer {
+  animation: flash 1.2s ease-in-out 0.25s;
+  background-color: #111111 !important;
+}
```

Line 2075 is overwriting [lines 549-551](https://github.com/pi-hole/AdminLTE/blob/72662faf34c44d6c0acd2ebe18270ce99db4ce39/style/themes/lcars.css#L549-L551).

Also, you used just `.user-header` and added `! important` to force the background change.

Please remove the `! important` and change the selectors to:
```suggestion
.navbar-nav > .user-menu > .dropdown-menu > .user-header,
.navbar-nav > .user-menu > .dropdown-menu > .user-footer {
  animation: flash 1.2s ease-in-out 0.25s;
  background-color: #111111;
}
```
Then move this block to line 549 (removing the old block).